docs/_documentations/codewind-eclipse-quick-guide.md

```diff
@@ -0,0 +1,167 @@
+title: Codewind in Eclipse
+description: Take advantage of Codewind's tools to help build high quality cloud native applications regardless of which IDE or language you use.
+keywords: Codewind, VS Code, microservice
+duration: 5 minutes
+permalink: codewind-eclipse-quick-guide
+---
+
+# Getting Started with Codewind in Eclipse 
+
+## Objectives
+* Develop a simple microservice, using Eclipse Codewind on Eclipse
+* Install VS Code and Codewind 
+
+## Overview 
+Eclipse Codewind provides the ability to create application projects from these `Application Stacks` that your company has built, enabling developers to focus on their code and not infrastructure and Kubernetes.  Application deployments to Kubernetes occur via pipelines when developers commit their local code to the correct Git repos Kabanero is managing via webhooks.    
+
+Eclipse Codewind provides the ability to create projects based on a variety of different template types.  These include IBM Cloud starters, OpenShift Do (odo), and Appsody templates. Today, there are templates for: IBM Cloud Starters, odo, Eclipse MicroProfile/Java EE, Springboot, Node.js, Node.js with Express, Node.js with Loopback.
+
+## Developing with Eclipse 
+If you use Eclipse, you can use Codewind for Eclipse to develop and debug your containerized projects from within a local Eclipse IDE.
+
+### Prerequisite
+Before you can develop a microservice with Eclipse, you need to:
+
+* [Install Docker](https://docs.docker.com/install/) 
+    * **Note:** Make sure to install or upgrade to minimum Docker version 19.03. 
+* [Install Eclipse](https://www.eclipse.org/downloads/packages/release/)
+    * **Note:** Make sure to install or upgrade to mimimum Eclipse version 2019-09 R (4.13.0). 
+
+### Installing Codewind for Eclipse
+The Codewind installation pulls the following images that form the Codewind backend:
+
+1. `eclipse/codewind-performance-amd64`
+2. `eclipse/codewind-pfe-amd64`
+
+The Codewind installation includes two parts:
+
+1. The Eclipse plug-in installs when you install Codewind from the [Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind)or when you install by searching in the `Eclipse Extensions` view.
+2. The Codewind backend containers install after you click `Install`. Clicking `Install` downloads the Codewind backend containers, ~1GB. 
+
+### Configuring Codewind to use application stacks
+Configure Codewind to use Appsody templates so you can focus exclusively on your code. These templates include an Eclipse MicroProfile stack that you can use to follow this guide. Complete the following steps to select the Appsody templates:
+
+1. Click the `Codewind` tab. 
+2. Expand `Codewind` by clicking the drop-down arrow.
+3. Right-click `Local [Running]`.
+4. Select `Manage Template Sources...`. 
+5. Select `Appsody Stacks - incubator`.
+6. Click the `OK` button. 
+
+You now have configured Codewind to Appsody templates and can proceed to develop your microservice within Codewind.
```

Comment:
`You now have` => `You have now...` ...   maybe `You have now configured Codewind to use Appsody templates and can ...`